cci_nepal/utils/dummy_data/dummy_data_utils.py

```diff
@@ -0,0 +1,69 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:light
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.13.6
+#   kernelspec:
+#     display_name: cci_nepal
+#     language: python
+#     name: cci_nepal
+# ---
+
+# %load_ext nb_black
+
+
+def label_transformer(d):
+
+    """
+
+    Takes a numerical score and assigns that back to categorical labels.
```

Comment:
Really small thing but I dont think you need the spaces in between the lines for the doc-string